packages/grid/x-data-grid-pro/src/DataGridProVirtualScroller.tsx

```diff
@@ -82,6 +93,21 @@ const getOverlayAlpha = (elevation: number) => {
   return alphaValue / 100;
 };
 
+const VirtualScrollerDetailPanels = styled('div', {
+  name: 'MuiDataGrid',
+  slot: 'DetailPanels',
```

Comment:
Should we document these 2 slots (ignore if it has already been done but I wasn't able to find it)?